src/components/Range.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  label {
+    margin-right: 10px;
+  }
+  output {
+    background: white;
+    margin: 10px;
+    padding: 6px 10px;
+    border-radius: 6px;
+    border: 1px solid ${({ theme }) => theme.border};
+  }
+`;
+const Input = styled.input`
+  align-items: center;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: none;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  &::-webkit-progress-bar {
+    background-color: red;
+  }
+  &::-webkit-slider-thumb {
+    height: 22px;
+    width: 22px;
+    border-radius: 50%;
+    align-self: center;
+    background: ${({ theme }) => theme.button};
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -10px;
+  }
+  &::-moz-range-thumb {
+    height: 22px;
+    width: 22px;
+    border-radius: 50%;
+    align-self: center;
+    background: ${({ theme }) => theme.button};
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -10px;
+    border: none;
+  }
+  &::-webkit-slider-runnable-track {
+    background: ${({ theme }) => theme.borderToggle};
+    height: 2px;
+  }
+
+  &::-moz-range-track {
+    background: ${({ theme }) => theme.borderToggle};
+  }
+  &::-moz-range-progress {
+    background-color: ${({ theme }) => theme.footerBackground};
+  }
+  &:focus {
+    outline: none;
```

Comment:
![we don't do that here](https://user-images.githubusercontent.com/3144128/68667076-0dbf3480-053d-11ea-99b1-8f9e62c052ac.gif)

You should [never remove focus outlines](https://a11yproject.com/posts/never-remove-css-outlines/).